lib/tasks/db/update_annotation_line.rake

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+namespace :db do
+  task update_annotation_line: :environment do
+    Rails.application.eager_load!
+
+    require 'action_view'
+    require 'action_view/helpers'
+
+    helper = HelperContext.new
+
+    start_date_input, end_date_input = ENV['START_DATE'], ENV['END_DATE']
```

Comment:
Instead of using environment variables like this, can we use `ARGV`, or user input like this https://github.com/Coursemology/coursemology2/blob/58e4b10095d7669f607eb616d8e0949ac5550fd9/lib/tasks/db/delete_phantom_course_users.rake#L6 instead